regression/ansi-c/goto_convert/invalid_goto_label/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE gcc-only
```

Comment:
Why is this gcc-only?

Reply:
You're right, this is a mistake, both `clang` and `gcc` report the same error.